cleverhans/future/torch/attacks/carlini_wagner_l2.py

```diff
@@ -0,0 +1,205 @@
+"""The CarliniWagnerL2 attack."""
+import torch
+
+
+INF = float('inf')
+def carlini_wagner_l2(model_fn, x, n_classes,
```

Comment:
This is a good point. I was abiding by the Zen of Python and trying to be explicit rather than implicit. But if @npapernot or @carlini think it's better to infer the number of classes from logits, I'm happy to make the change. Thanks.